packages/salesforcedx-slds-linter/package.json

```diff
@@ -38,12 +38,13 @@
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run test:vscode-integration",
     "test:vscode-integration":
-      "node ../../scripts/run-vscode-integration-tests",
+      "node ../../scripts/run-vscode-integration-tests-with-top-level-extensions",
```

Comment:
One thing I just noticed is that changing this to run with `run-vscode-integration-test-with-top-level-extensions` will install all the other extensions in the vscode test instance. Do we need that for this ?

Reply:
Ah okay, so it looks like this test does need a vscode instance, but it doesn't need all the other extensions for the test. In that case, should I reverse this change?